src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -1,14 +1,80 @@
+#define END_INTERACTION 2
+
 void EnOssan_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnOssan_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnOssan_Update(Actor* thisx, GlobalContext* globalCtx);
 
-#if 0
+void EnOssan_DrawCuriosityShopMan(Actor* thisx, GlobalContext* globalCtx);
+void EnOssan_DrawPartTimeWorker(Actor* thisx, GlobalContext* globalCtx);
+
+void EnOssan_InitCuriosityShopMan(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_InitPartTimeWorker(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_InitShop(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_Idle(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_BeginInteraction(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_Hello(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_SetHaveMet(EnOssan* this);
+void EnOssan_StartShopping(GlobalContext* globalCtx, EnOssan* this);
+void EnOssan_FaceShopkeeper(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_LookToShopkeeperFromShelf(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_TalkToShopkeeper(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_LookToLeftShelf(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_LookToRightShelf(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_BrowseLeftShelf(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_BrowseRightShelf(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_SelectItem(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_BuyItemWithFanfare(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_CannotBuy(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_CanBuy(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_SetupItemPurchased(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_ItemPurchased(EnOssan* this, GlobalContext* globalCtx);
+void EnOssan_ResetItemPosition(EnOssan* this);
+void EnOssan_Blink(EnOssan* this);
+void EnOssan_GetCutscenes(EnOssan* this, GlobalContext* globalCtx);
+
+extern UNK_TYPE D_0401F740;
+extern UNK_TYPE D_0401F8C0;
+extern UNK_TYPE D_0401F7C0;
+extern FlexSkeletonHeader D_06013320;
+extern AnimationHeader D_06012C34;
+extern FlexSkeletonHeader D_060028A0;
+extern AnimationHeader D_06009D34;
+extern AnimationHeader D_0600CB3C;
+extern AnimationHeader D_0600DE34;
+extern AnimationHeader D_0600E3EC;
+extern AnimationHeader D_0600C58C;
+extern AnimationHeader D_0600F00C;
+extern AnimationHeader D_0600B9D8;
+extern AnimationHeader D_0600C26C;
+extern AnimationHeader D_0600D354;
+extern AnimationHeader D_060131FC;
+extern AnimationHeader D_0601430C;
+extern AnimationHeader D_060138B0;
+extern AnimationHeader D_0600A460;
+extern UNK_TYPE D_06006498;
+extern UNK_TYPE D_06007140;
+extern UNK_TYPE D_06006D40;
+extern UNK_TYPE D_06005BC0;
+extern UNK_TYPE D_06006F18;
+extern UNK_TYPE D_06006B18;
+
+const char filename1[] = "../z_en_oB1.c"; // Unused
+const char filename2[] = "../z_en_oB1.c"; // Unused
+const char filename3[] = "../z_en_oB1.c"; // Unused
```

Comment:
That does in fact match. 
There are three functions that were basically copy pasted from OOT that would have referenced the filename, so feel they would best put there (`EnOssan_DrawCursor`, `EnOssan_DrawTextRec`, `EnOssan_DrawStickDirectionPrompts`) right under declarations. If there is a better place for them please let me know.

Reply:
I think best to leave them at the top until we work out exactly how we want to deal with them. I think the order doesn't matter much since there's no other rodata like this in the actor, although I may be mistaken.

Reply:
In order to use `((void)"Unreferenced String")`, I have to put it in some function and cannot leave it at the top (syntax errors). However, it does not matter which functions I put them in, or how many in one function. If we want them at the top I would have to go back to `const char filename[]`.